.github/workflows/build_and_test.yml

```
@@ -1500,21 +1492,16 @@ jobs:
   #       cat "$GITHUB_WORKSPACE/OpenDDS/${{ github.job }}_autobuild_workspace/latest.txt"
   #       if ! grep -q 'Failures: 0' "$GITHUB_WORKSPACE/OpenDDS/${{ github.job }}_autobuild_workspace/latest.txt"; then echo "::error file=latest.txt,line=0,col=0::Test Failures: see 'Test Results: ${{ github.job }}'"; fi
 
-  ACE_TAO_u18_gcc6_d0w1_cpp03:
+  ACE_TAO_u18_gcc6_d0w1_cpp03_2:
 
     runs-on: ubuntu-18.04
 
     needs: wait_cancel_previous
 
     steps:
-    - name: install gcc
+    - name: install gcc and g++
       run: |
-        sudo apt-get install -y gcc-6
-        sudo update-alternatives \
-          --install /usr/bin/gcc gcc /usr/bin/gcc-6 100 \
-          --slave /usr/bin/gcc-ar gcc-ar /usr/bin/gcc-ar-6 \
-          --slave /usr/bin/gcc-ranlib gcc-ranlib /usr/bin/gcc-ranlib-6 \
-          --slave /usr/bin/gcov gcov /usr/bin/gcov-6
+        sudo apt-get install gcc-6 g++-6
```

```suggestion
        sudo apt-get install g++-6
```

We don't use the gcc C compiler, although it will get installed anyway because the g++-X packages depend on the corresponding gcc-X packages.

---

FYI libACE does use `$(CC)` in place of `$(CXX)` for a few translation units.
I agree with letting the apt dependencies sort it out.

---

Huh, I have seen the C support in gnuace, but I didn't think it was actually using it.